src/Mgate/SuiviBundle/Resources/views/Etude/Tab/EtudesAvortees.html.twig

```diff
@@ -1,3 +1,4 @@
+<div class="table-responsive">
```

Comment:
Allow an horizontal scroll on mobile / small screens. Not the best solution but it's always better than not have access at it.